js/interface.js

```diff
@@ -838,6 +1089,115 @@ $('.file-manager-wrapper')
   })
   .on('hide.bs.collapse', '.panel-collapse', function() {
     $(this).siblings('.panel-heading').find('.fa').removeClass('rotate');
+  })
+  .on('hidden.bs.modal', '#move-modal', function() {
+    $(this).remove();
+    foldersForMoving = {};
+  })
+  .on('change', '#move-model-organizations-select', function() {
+    $('.move-modal-empty-state').removeClass('active');
+    $('[data-move-button]').attr('disabled');
+    foldersForMoving['navStack'] = [];
+    var type = $(this).find(":selected").attr('data-type');
```

Comment:
please stick to single quotes whenever possible

Reply:
fixed